org.eclipse.xtext.xtext.ui.examples/projects/fowlerdsl/org.eclipse.xtext.example.fowlerdsl.ui.tests/src/org/eclipse/xtext/example/fowlerdsl/ui/tests/StatemachineHighlightingTest.xtend

```diff
@@ -0,0 +1,182 @@
+package org.eclipse.xtext.example.fowlerdsl.ui.tests
+
+import org.eclipse.swt.SWT
+import org.eclipse.xtext.testing.InjectWith
+import org.eclipse.xtext.testing.XtextRunner
+import org.eclipse.xtext.ui.testing.AbstractHighlightingTest
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(XtextRunner)
+@InjectWith(StatemachineUiInjectorProvider)
+class StatemachineHighlightingTest extends AbstractHighlightingTest {
+
+	@Test def events_keyword() {
+		'''
+			events
+				doorClosed   D1CL
+				drawerOpened D2OP
+				lightOn      L1ON
+				doorOpened   D1OP
+				panelClosed  PNCL
+			end
+		'''.testHighlighting("events", SWT.BOLD, 127, 0, 85)
```

Comment:
My gut feeling is, that we'd rather like to check that a certain style is applied from the highlighting configuration instead of checking the colors directly.